Tests/XcresultparserTests/XcresultparserTests.swift

```diff
@@ -166,4 +177,16 @@ Summary
         sut = OutputFormat(string: "xyz")
         XCTAssertEqual(OutputFormat.cli, sut)
     }
+    
+    // MARK: helper functions
+    
+    func assertXmlTestReportsAreEqual(expectedFileName: String, actual: XmlSerializable) {
+        
+        let expectedResultFile =  Bundle.module.url(forResource: expectedFileName, withExtension: "xml")!
+
+        let actualXMLDocument = try! XMLDocument.init(data: Data("\(actual.xmlString)\n".utf8), options: [])
```

Comment:
Since this nice support of throwing test functions was added to the XCTest framework, we should take advantage of it and just make assertXmlTestReportsAreEqual() a throwing function and call it with "try", instead crashing the tests here with the force unwrap.